python/ql/src/semmle/python/AstExtended.qll

```diff
@@ -3,6 +3,9 @@ import python
 /** Syntactic node (Class, Function, Module, Expr, Stmt or Comprehension) corresponding to a flow node */
 abstract class AstNode extends AstNode_ {
 
+    /* Special comment for documentation generation */
```

Comment:
What purpose does this comment serve? It only appears in _some_ cases, and I don't immediately see what the pattern is.

Reply:
This is the root of the heirarchy. So it should only be here. I've clarified this one and deleted the others.